documentation/config_file_puppetserver.markdown

```diff
@@ -94,6 +96,8 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 jruby-puppet: {
     ruby-load-path: [/opt/puppetlabs/puppet/lib/ruby/vendor_ruby]
     gem-home: /opt/puppetlabs/server/data/puppetserver/jruby-gems
+    environment-vars: { "HOME" : ${HOME}
+                        "LANG" : "de_DE.UTF-8" }
```

Comment:
I probably wouldn't use HOME since we happen to have special-cased that one at times.